lib/emit.generated.js

```diff
@@ -289,40 +354,51 @@ const imports = {
           const a = state0.a;
           state0.a = 0;
           try {
-            return __wbg_adapter_23(a, state0.b, arg0, arg1);
+            return __wbg_adapter_25(a, state0.b, arg0, arg1);
           } finally {
             state0.a = a;
           }
         };
-        const ret = new Promise(cb0);
+        var ret = new Promise(cb0);
         return addHeapObject(ret);
       } finally {
         state0.a = state0.b = 0;
       }
     },
     __wbg_resolve_a9a87bdd64e9e62c: function (arg0) {
-      const ret = Promise.resolve(getObject(arg0));
+      var ret = Promise.resolve(getObject(arg0));
       return addHeapObject(ret);
     },
     __wbg_then_ce526c837d07b68f: function (arg0, arg1) {
-      const ret = getObject(arg0).then(getObject(arg1));
+      var ret = getObject(arg0).then(getObject(arg1));
```

Comment:
I think you are using an outdated version of `wasm-bindgen-cli`

Reply:
Updated. Would be nice if we implemented https://github.com/denoland/wasmbuild/issues/6 soon.